tasks/configure.yml

```diff
@@ -80,6 +80,18 @@
     - mysql_log_error | default(false)
   tags: ['skip_ansible_galaxy']
 
+- name: Initialize mysql when mysql_datadir is customized (MySQL)
+  ansible.builtin.command: mysqld --initialize-insecure --user=mysql --datadir={{ mysql_datadir }}
```

Comment:
This could interfere with later initialization, though, no?

Reply:
Which later initialization? I can't get molecule to run the rockylinux on my m1 mac, but the ubuntu2204 and 2004, both seem to pass the tests, when specifying an alternate data directory for both mysql and mariadb. 


I am trying to modify the github actions to run the additional tests (I believe that is the way to do it?) I can't find a way to have different playbooks in molecule that tear down everything between playbooks.

Reply:
It still depends. @geerlingguy do you want to have an extra molecule scenario for a moved directory or should it be part of `default`? I made a PR to the PR of @misilot and I'm ready to change as you want it to have.

Reply:
So, I am working on the logic to check the version number instead of looking for package names, since debian uses mysql in the package name, but is really mariadb, the logic I have doesn't work

Reply:
Ok, I fixed the logic, and they all seem to run locally with molecule. I also added in @widhalmt change to this PR.